internal/executor/container_examples/egress_server/cmd/server/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type Result struct {
+	Name    string
+	Success bool
+}
+
+func main() {
+	r := mux.NewRouter()
+	r.HandleFunc("/experiment/CheckEgress/", CheckEgress)
+
+	// Start the experiment server
+	log.Print("starting server on :4000")
+	err := http.ListenAndServe(":4000", r)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+}
+
+func CheckEgress(w http.ResponseWriter, r *http.Request) {
+	endpoints := []string{
+		"https://google.com",
```

Comment:
might be good to have these test URLs in the experiment config and we pass this in as part of the experiment verify